R/utilities-sensitivity-analysis.R

```diff
@@ -585,14 +616,21 @@ plotPopulationSensitivity <- function(structureSets,
   dFList <- list()
   i <- 0

```

Comment:
I think `i` is not useful here, it would be better to use the pk parameter name as the list element of dFList line 654

Reply:
Fixed